ThirdParty/helics/external/CLI11/CLI11.hpp

```diff
@@ -5233,6 +5734,54 @@ class App {
         return help_all_ptr_;
     }
 
+    /// Set a version flag and version display string, replace the existing one if present
+    Option *set_version_flag(std::string flag_name = "", const std::string &version = "") {
+        // take flag_description by const reference otherwise add_flag tries to assign to help_description
+        if(version_ptr_ != nullptr) {
+            remove_option(version_ptr_);
+            version_ptr_ = nullptr;
+        }
+
+        // Empty name will simply remove the help flag
+        if(!flag_name.empty()) {
+            version_ptr_ = add_flag_callback(flag_name,
+                                             [version]() { throw(CLI::CallForVersion(version, 0)); },
+                                             "display program version information and exit");
+            version_ptr_->configurable(false);
+        }
+
+        return version_ptr_;
+    }
+    /// Set a version flag, using a callback
+    Option *set_version_flag(std::string flag_name, std::function<std::string()> vfunc) {
+        // take flag_description by const reference otherwise add_flag tries to assign to help_description
+        if(version_ptr_ != nullptr) {
+            remove_option(version_ptr_);
+            version_ptr_ = nullptr;
+        }
+
+        // Empty name will simply remove the help flag
+        if(!flag_name.empty()) {
+            version_ptr_ = add_flag_callback(flag_name,
+                                             [vfunc]() { throw(CLI::CallForVersion(vfunc(), 0)); },
+                                             "display program version information and exit");
+            version_ptr_->configurable(false);
+        }
+
+        return version_ptr_;
+    }
+    /// Set the version as a string
+    App *version(const std::string &version) {
+        set_version_flag("-v,--version", version);
```

Comment:
I'm don't think `-v` should be "reserved" by default. It's used for verbose by a lot of commonly used commands (rm, cp, mv, curl, wget).